src/platform_impl/web/web_sys/scaling.rs

```diff
@@ -58,10 +58,12 @@ impl ScaleChangeDetectorInternal {
         // This media query initially matches the current `devicePixelRatio`.
         // We add 0.0001 to the lower and upper bounds such that it won't fail
         // due to floating point precision limitations.
+        let min_scale = current_scale - 0.0001;
+        let max_scale = current_scale + 0.0001;
         let media_query = format!(
-            "(min-resolution: {:.4}dppx) and (max-resolution: {:.4}dppx)",
-            current_scale - 0.0001,
-            current_scale + 0.0001,
+            "(min-resolution: {:.4}dppx) and (max-resolution: {:.4}dppx),
+             (-webkit-min-device-pixel-ratio: {:.4}) and (-webkit-max-device-pixel-ratio: {:.4})",
+            min_scale, max_scale, min_scale, max_scale,
```

Comment:
Rather than repeating `min_scale` and `max_scale`, could you use the named format mode? e.g. `{min_scale:.4}`. It should make this a little clearer

Reply:
Ah yes, excellent point, that's much cleaner.